extensions/amp-story-auto-ads/0.1/test/test-amp-story-auto-ads.js

```diff
@@ -502,5 +502,18 @@ describes.realWin(
         );
       });
     });
+
+    describe('Test if Handle Config assigns a new conifg and returns a promise', async () => {
+      const handleConfig = await new this.handleConfig_();
```

Comment:
Lets try not to call private methods in tests if we can avoid it. It's not really the responsibility of this class to parse the config, but to take the returned config and create an <amp-ad> from it. If you want to test that you can do something like call `layoutCallback` and make sure the amp-ad is in the dom with the right attrs from the config (this test may already exist).